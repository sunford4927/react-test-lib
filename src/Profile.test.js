import React from "react";
import { render } from "@testing-library/react";
import Profile from "./Profile";

describe("<Profile />", () => {
    it("matches snapshot", () => {
        const utils =render(<Profile username="lsh1012" name="이상현" />);
        expect(utils.container).toMatchSnapshot();
    });
    it("shows the props correctly", () => {
        const utils = render(<Profile username="lsh1012" name="이상현" />);
        utils.getByText("lsh1012");
        utils.getByText("(이상현)");
        // utils.getByText("lsh1012(이상현)");
    });
});