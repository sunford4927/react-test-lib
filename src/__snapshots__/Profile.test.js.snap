// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Profile /> matches snapshot 1`] = `
<div>
  <div>
    <b>
      lsh1012
    </b>
    <span>
      (
      이상현
      )
    </span>
  </div>
</div>
`;
